xformers/triton/utils.py

```diff
@@ -44,7 +44,25 @@ def assert_almost_equal(x, y, decimal=2, err_msg=""):
     import numpy.testing as npt
 
     if isinstance(x, torch.Tensor):
+        if x.dtype == torch.bfloat16:
+            x = x.float()
```

Comment:
Wouldn't it be better to modify the test code to perform the appropriate castings, instead of changing the assert function? Also, IIRC this function now has been integrated in triton and removed in a PR from @blefaudeux that I've just merged

Reply:
I think that it's better to use the triton helper here, we're trusting triton in this part of the stack anyway

Reply:
Makes sense, I removed this and `assert_allclose`